examples/csharp/src/services/database.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections.Generic;
+using Service;
+
+public class Database : Service
+{
+    public ListCollections(string search = "", int limit = 25, int offset = 0, string orderType = "ASC")
+    {
+        paramsIn = new Dictionary<string, string>
+        {
+            "search" = search,
+            "limit" = limit,
+            "offset" = offset,
+            "orderType" = orderType
+        };
+        path = $"/database";
+
+        return _client.Call("get", path, new Dictionary<string, string>(), paramsIn);
+    }
+
+    public CreateCollections(string name = "", string[] read = "", string[] write = "", string[] rules = "")
```

Comment:
Should this be called CreateCollection() ? I think this method only creates a single collection, correct? Same for UpdateCollections, GetCollections, DeleteCollections. The operations for documents have correct names i.e. DeleteDocuments and ListDocuments :)